readme.md

```diff
@@ -290,12 +291,12 @@ See the [ESLint docs](https://eslint.org/docs/user-guide/configuring/configurati
 
 ## Maintainers
 
-- [Sindre Sorhus](https://github.com/sindresorhus)
-- [Adam Babcock](https://github.com/MrHen)
-- [futpib](https://github.com/futpib)
-- [Fisker Cheung](https://github.com/fisker)
+* [Sindre Sorhus](https://github.com/sindresorhus)
```

Comment:
I prefer `-` for lists.

Reply:
Updated this file to use dash `-` lists.